arc_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 )
 
+//nolint:gochecknoinits
```

Comment:
Rather than do this can you just not include the noinit checker in the config?

Reply:
Fixed, please take a look once more.